scripts/ci

```diff
@@ -354,9 +356,18 @@ def dpkg_binary(dsc_path, name, git, series, build_arch, build_all):
                 "--extra-repository=deb http://ppa.launchpad.net/" + ppa_proposed + "/ubuntu " + series.codename + " main",
                 "--extra-repository=deb-src http://ppa.launchpad.net/" + ppa_proposed + "/ubuntu " + series.codename + " main",
                 "--extra-repository-key=" + path.join(POP_DIR, "scripts", ppa_key),
+                "--extra-repository-key=" + path.join(POP_DIR, "scripts", ".pop-os.asc"),
                 "--no-apt-distupgrade",
             ]
 
+            for (repo, _) in pockets:
```

Comment:
My main concern here is what might be in `pockets`, and the fact that this potentially adds multiple additional repositories. If I understand, if the same commit is pushed to multiple branches, the CI will use the same build. I'm not sure if the dependencies it adds here might then cause possible problems...

Reply:
All you really need is to add the branch the package is being built from, so if building for focal on branch `new-feature`, you'd add `http://apt.pop-os.org/staging/new-release focal main`.

Reply:
My understanding is that in the (unlikely) case that the same commit is pushed to two branches between CI runs, `pockets` will contain both branches, and the same build will be used for both. If another branch is later updated to a commit that already has a build for another branch (for instance, a fast-forward merge), the package will be copied without rebuilding.

That's why this requires a loop and potentially could add multiple branches. But I'm happy to be corrected.